### DISCUSSION TITLE
Traverse compat addon

### DISCUSSION THREAD

**Discussion by roaxyane** [DynamicTreesTeam/DynamicTrees#961] [Compatibility Addon Suggestions]
traverse mod: https://www.curseforge.com/minecraft/mc-mods/traverse

there is a dynamic tree addon for traverse but latest supported ver is 1.18 https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-traverse

**Comment by supermassimo** [comment]
The addon has now been updated to 1.19 and 1.20 https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-traverse
